fixes/github-issue-create.js

```diff
@@ -169,53 +207,100 @@ async function getTopCommittersOfRepository(targetOrg, targetRepository) {
 }
 
 /**
- * Create an issue on Github with labels and all on the target repository.
+ * Create an issue on GitHub with labels and all on the target repository.
  *
  * @param {object} options The rule configuration.
- * @returns {object} Returns issue after adding it via the Github API.
+ * @param {number} assigneeSelectCount counter for which assignee was selected
+ * @param {array<object>} assignees array of contributors
+ * @returns {object} Returns issue after adding it via the GitHub API.
  */
-async function createIssueOnGithub(options) {
-  try {
-    return await this.Octokit.request('POST /repos/{owner}/{repo}/issues', {
-      owner: targetOrg,
-      repo: targetRepository,
-      title: options.issueTitle,
-      body: InternalHelpers.generateIssueBody(options),
-      labels: options.issueLabels,
-      assignees: issuesAssignees
-    })
-  } catch (e) {
-    console.error(e)
-  }
+async function createIssueOnGithub(options, assigneeSelectCount, assignees) {
```

Comment:
Consider changing signature to `createIssueOnGithub(options, assignees, assigneeSelectIndex = 0)` and changing line 169 to `const newIssue = await createIssueOnGithub(options, assignees)` and changing line 243 to `return createIssueOnGithub(options, assignees, assigneeSelectCount)`.